packages/block-library/src/image/image.js

```diff
@@ -94,6 +102,216 @@ const ImageWrapper = ( { href, children } ) => {
+										<>
+											<ExternalLink
+												href={
+													// translators: Localized tutorial, if one exists. W3C Web Accessibility Initiative link has list of existing translations.
+													__(
+														'https://www.w3.org/WAI/tutorials/images/decision-tree/'
+													)
+												}
+											>
+												{ __(
+													'Describe the purpose of the image.'
+												) }
+											</ExternalLink>
+											<br />
+											{ __(
+												'Leave empty if decorative.'
+											) }
+										</>
+									)
+								}
+								__nextHasNoMarginBottom
+							/>
+						) }
+					/>
+				) }
+				{ activeControls.includes( 'title' ) && (
+					<Dropdown
+						defaultOpen={
+							! previousActiveControls?.includes( 'title' )
+						}
+						popoverProps={ WRITEMODE_CONTROLS_POPOVER_PROPS }
+						renderToggle={ ( { isOpen, onToggle } ) => (
+							<ToolbarButton
+								onClick={ onToggle }
+								aria-haspopup="true"
+								aria-expanded={ isOpen }
+								onKeyDown={ ( event ) => {
+									if ( ! isOpen && event.keyCode === DOWN ) {
+										event.preventDefault();
+										onToggle();
+									}
+								} }
+							>
+								{ __( 'Title' ) }
+							</ToolbarButton>
+						) }
+						renderContent={ () => (
+							<TextControl
+								__next40pxDefaultSize
+								className="wp-block-image__toolbar_content_textarea"
+								__nextHasNoMarginBottom
+								label={ __( 'Title attribute' ) }
+								value={ attributes.title || '' }
+								onChange={ ( value ) =>
+									setAttributes( { title: value } )
+								}
+								disabled={ lockTitleControls }
+								help={
+									lockTitleControls ? (
+										<>{ lockTitleControlsMessage }</>
+									) : (
+										<>
+											{ __(
+												'Describe the role of this image on the page.'
+											) }
+											<ExternalLink href="https://www.w3.org/TR/html52/dom.html#the-title-attribute">
+												{ __(
+													'(Note: many devices and browsers do not display this text.)'
+												) }
+											</ExternalLink>
+										</>
+									)
+								}
+							/>
+						) }
+					/>
+				) }
+			</BlockControls>
+		);
+	}
+
+	return (
+		// Add some extra controls for content attributes when content only mode is active.
+		// With content only mode active, the inspector is hidden, so users need another way
+		// to edit these attributes.
+		<>
+			{ controls }
+			<BlockControls group="inline">
+				<ToolbarItem>
+					{ ( toggleProps ) => (
+						<DropdownMenu
+							icon={ chevronDown }
+							/* translators: button label text should, if possible, be under 16 characters. */
+							label={ __( 'More' ) }
+							toggleProps={ {
+								...toggleProps,
+								disabled:
+									activeControls.length ===
+									attributesToDisplayControls.length,
+								accessibleWhenDisabled: true,
+								// TODO: update this one..
+								description: __( 'Displays more tools…' ),
```

Comment:
We need to find a proper description here.

Reply:
I'd remove the ellipsis, but other than that this matches what we do `FormatToolbar`, which has been worked on by a11y experts.

We _could_ try to be more descriptive (i.e. find something accurate and concise like "display more content-editing controls", etc.), but in the end couldn't that verbosity get in the way of the user? The sooner they'll expand the dropdown menu, the sooner they can get a true sense of what hides in there.

(Precise and verbose descriptions can also get in the way of translation.)